services/openshift/scripts/add_insecure_registry

```diff
@@ -36,7 +36,8 @@ function init_insecure_reg()
         sed -i.orig "/# INSECURE_REGISTRY=*/c\INSECURE_REGISTRY=\"\"" ${docker_conf_file}
         # Get machine IP address
         local local_ip=$(get_ip_address)
-        sed -i.back "s/INSECURE_REGISTRY=\"\(.*\)\"/INSECURE_REGISTRY=\"\1 --insecure-registry 172.30.0.0\/16 --insecure-registry $local_ip\"/" ${docker_conf_file}
+        local registry_host_name="hub.openshift.${local_ip}.xip.io"
```

Comment:
> Is the problem not in openshift_option. There we just use the hostname, but it should be something like 10.1.2.2.xip.io or $(hostname).10.1.2.2.xip.io. In the latter case the hub host would be something like hub.openshift.$(hostname).10.1.2.2.xip.io.

We are not keeping 10.1.2.2.xip.io as host in openshift_option because than it  will make `10.1.2.2` as hard coded IP and we again have issue if a user use other than `10.1.2.2` and openshift service can't take those env variable dynamically so that's why we are using it that way. In future we can include it as part of `sccli` which rewrite `openshift_option` as per passed env.